bindings/nodejs/inc/nodejs_common.h

```diff
@@ -55,6 +56,7 @@ struct NODEJS_MODULE_HANDLE_DATA
     NODEJS_MODULE_HANDLE_DATA(NODEJS_MODULE_HANDLE_DATA&& rhs)
     {
         broker = rhs.broker;
+        create_complete = std::move(rhs.create_complete);
```

Comment:
Regardless of the life of any individual data structure used to contain the handle data, the collection of handle data represents a single object. So I believe the future (along with all other data) should be moved if possible to reflect the intent of what is happening.
Unfortunately, we are only able to preserve this data in the MOVE constructor and not in the copy constructors. I see how this is more confusing than not bring them along all together, but I must admit I don't like the way it feels.